linera-core/src/remote_node.rs

```diff
@@ -123,7 +123,10 @@ impl<N: ValidatorNode> RemoteNode<N> {
             let certificates = self
                 .node
                 .download_certificates(info.requested_sent_certificate_hashes)
-                .await?;
+                .await?
+                .into_iter()
+                .map(|c| c.try_into().expect("Expected ConfirmedBlock certificate"))
```

Comment:
In order to _not_ change the gRPC API, I'm mapping back to the `Certificate` type. `download_certificates` comes from the `ValidatorNode` trait which then is implemented for the protobuf service.

Reply:
Hmm, I don't know how I missed that. Thanks.